### PR TITLE
[bug 1254619] update code for django 1.9

### PR DESCRIPTION
**Pull request by Mte90**

### CODE REVIEW

File: mozillians/users/tests/test_models.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from django.core.urlresolvers import reverse
 from django.db.models.query import QuerySet
 from django.test import override_settings
-from django.utils import unittest
+import unittest
```
Comment (akatsoulas) [inline mozillians/users/tests/test_models.py]: Could you please move this line above the django `from` statements? This should be right above line 3.

**Comment by akatsoulas** [comment]
@Mte90 I believe we are only missing the last warning from the referenced bug and we should be good to go.

**Comment by Mte90** [comment]
done and merged in one commit :-)

File: mozillians/users/managers.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
-from django.db.models import Q, Manager, get_model
+from django.db.models import Q, Manager
+from django.apps import apps
```
Comment (akatsoulas) [inline mozillians/users/managers.py]: @Mte90 can you move this above the `django.db.models` ?

File: mozillians/users/tests/test_models.py [hunk 3]
```diff
@@ -1,13 +1,13 @@
 # -*- coding: utf-8 -*-
 from datetime import datetime
 from uuid import uuid4
+import unittest
```
Comment (akatsoulas) [inline mozillians/users/tests/test_models.py]: Please move this right above `from datetime import datetime`.

**Comment by Mte90** [comment]
done

**Comment by akatsoulas** [comment]
Thanks @Mte90 r+ 🚀

**Comment by Mte90** [comment]
merged all the commits again

**Pull request merged**